src/engine/source/api/include/api/config/config.hpp

```diff
@@ -135,11 +135,11 @@ api::Handler runtimeSave(ConfHandler<ConfDriver> confHandler)
 }
 
 template<typename ConfDriver>
-bool registerHandlers(std::shared_ptr<api::Registry> registry, ConfHandler<ConfDriver> confHandler)
+bool registerHandlers(std::shared_ptr<api::Api> api, ConfHandler<ConfDriver> confHandler)
```

Comment:
Can this registerHandlers be change to void or to unify the others catalog, kvdb, config) to bool return type?

Reply:
done